src/main/java/org/testng/internal/Parameters.java

```diff
@@ -200,6 +257,21 @@ private static void checkParameterTypes(String methodName,
     }
   }
 
+  private static String prettyFormat(List<Class> classes) {
+    StringBuilder builder = new StringBuilder("<");
+    if (classes.size() == 1) {
```

Comment:
No need to check against size=1, the loop will work if the list size = 1 too.
But currently, the method will fail if the list is empty ;)